cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -36,6 +36,13 @@
     /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libudev.so* mr,
     /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libseccomp.so* mr,
     /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libcap.so* mr,
+    /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libm-[0-9]*.so* mr,
+    /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libdw-[0-9]*.so* mr,
```

Comment:
Curious, what is `libdw`?

Reply:
Guessing it is the "library that provides access to the DWARF debug information"